modules/installation/pages/install-server.adoc

```diff
@@ -40,12 +40,13 @@ image::unifiedinstaller-product.png[scaledwidth=80%]
 . In the [guimenu]``Registration`` screen, check the [guimenu]``Register System via scc.suse.com`` checkbox, enter your {scc} credentials, and click btn:[Next].
 . OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next].
 . In the [guimenu]``System Role`` screen, check the [guimenu]``SUSE Manager Server`` checkbox, and click btn:[Next].
-. In the [guimenu]``Suggested Partitioning`` screen, accept the default values, or use the btn:[Guided Setup] or btn:[Expert Practitioner] options to customize your partitioning model, and click btn:[Next].
+. In the [guimenu]``Suggested Partitioning`` screen, accept the default values, or use the btn:[Guided Setup] or btn:[Expert Partitioner] options to customize your partitioning model, and click btn:[Next].
```

Comment:
OMG, I have been reading that as "practitioner" all this time. I didn't believe you and had to go back and check! face->palm